packages/components/src/presentation-components/instance-filter-builder/PresentationInstanceFilter.ts

```diff
@@ -529,3 +546,20 @@ function isPoint3dLike(value: object): value is { x: number; y: number; z: numbe
 function isInstanceKeyLike(value: object): value is { id: string; className: string } {
   return (value as any).id !== undefined && (value as any).className !== undefined;
 }
+
+function createUniqueValue(value: PropertyValue): PrimitiveValue | undefined {
+  if (value.valueFormat !== PropertyValueFormat.Primitive || value.displayValue === undefined || value.value === undefined) {
+    return undefined;
```

Comment:
doesn't this mean that we were given an invalid `PrimitiveValue`? maybe we should throw instead?

Reply:
I guess we could throw if `valueFormat` is not `Primitive`. But as this is an utility function for creating conditions using unique values selector, I think makes it more convenient for consumer not throwing error.